Source/Scene/Gltf3DTileContent.js

```diff
@@ -144,6 +144,7 @@ function initialize(content, gltf) {
 
   if (ExperimentalFeatures.enableModelExperimental) {
     modelOptions.customShader = tileset.customShader;
+    modelOptions.content = content;
```

Comment:
Are we planning to apply these changes to B3DM + ModelExperimental?

Reply:
No, 1.0 formats are not in the scope of this PR.